packages/devtools-evm/src/signer/sdk.ts

```diff
@@ -130,23 +134,20 @@ export class GnosisOmniSignerEVM<TSafeConfig extends ConnectSafeConfig> extends
     }
 
     async #initSafe() {
-        if (this.safeConfig && (!this.safeSdk || !this.apiKit)) {
+        if (!this.safeSdk || !this.apiKit) {
             const ethAdapter = new EthersAdapter({
                 ethers,
                 signerOrProvider: this.signer,
             })
             this.apiKit = new SafeApiKit({ txServiceUrl: this.safeUrl, ethAdapter })
 
-            const contractNetworks = this.safeConfig.contractNetworks
             this.safeSdk = await Safe.create({
                 ethAdapter,
                 safeAddress: this.safeConfig.safeAddress!,
-                ...(!!contractNetworks && { contractNetworks }),
+                contractNetworks: this.safeConfig.contractNetworks,
             })
         }
-        if (!this.safeSdk || !this.apiKit) {
-            throw new Error('Safe SDK not initialized')
-        }
+
```

Comment:
log msg for already init?

Reply:
else case~

Reply:
might clog the debug if theres hundreds of tx though, so maybe not

Reply:
that's gonna log on every transaction, this method is being called for every `signAndSend`

Reply:
yeah realized that, nvm